README.md

```diff
@@ -2,7 +2,7 @@
 [![Travis-CI Build
 Status](https://travis-ci.org/ThinkR-open/remedy.svg?branch=master)](https://travis-ci.org/ThinkR-open/remedy)[![Coverage
 status](https://codecov.io/gh/ThinkR-open/remedy/branch/master/graph/badge.svg)](https://codecov.io/github/ThinkR-open/remedy?branch=master)[![Covrpage
-Summary](https://img.shields.io/badge/covrpage-Last_Build_2018_06_28-yellowgreen.svg)](https://github.com/ThinkR-open/remedy/tree/master/tests/README.md)
+Summary](https://img.shields.io/badge/covrpage-Last_Build_2018_06_29-yellowgreen.svg)](https://github.com/ThinkR-open/remedy/tree/master/tests/README.md)
```

Comment:
that date is for the covrpage build and not the current date

Reply:
🤔 I only knit README.Rmd so that must have been changed due to `sprintf('Last_Build_%s',strftime(Sys.Date(),format = '%Y_%m_%d'))` in the Rmd. Is there a way to avoid this, i.e. how do I update README.md without triggering that? Or does TRAVIS CI update README.md (so

Reply:
ah. yeah im updating covrpage locally to be manage that. that line in the rmd should be made static for now

Reply:
Ok, reverted that line.

Reply:
Sorry by the way, I should've seen that.